lib-injection/sitecustomize.py

```diff
@@ -38,7 +38,9 @@ def _configure_ddtrace():
         try:
             pkgs_path = os.path.join(os.path.dirname(__file__), "ddtrace_pkgs")
             subprocess.run(
-                [sys.executable, "-m", "pip", "install", "--find-links", pkgs_path, "ddtrace"], env=env, check=True
+                [sys.executable, "-m", "pip", "install", "--no-index", "--find-links", pkgs_path, "ddtrace"],
+                env=env,
+                check=True,
```

Comment:
we should have a regression test for this to ensure the install works for offline usage.

I tried modifying the lib_injection.yml unit tests to use a `--internal` network but that prevented the webserver port from being accessible outside the container

Reply:
Tests will be added in a follow up PR. I think we should get this fix out ASAP